services-custom/s3-transfer-manager/src/main/java/software/amazon/awssdk/transfer/s3/internal/DefaultS3TransferManager.java

```diff
@@ -100,18 +106,6 @@ private static TransferManagerConfiguration resolveTransferManagerConfiguration(
         return transferConfigBuilder.build();
     }
 
-    private static S3CrtAsyncClient initializeS3CrtClient(DefaultBuilder tmBuilder) {
```

Comment:
Should we still create a default client if no custom client is provided?

Reply:
added in ctor as mentioned in another comment